internal/nodes/avi_model_nodes.go

```diff
@@ -1077,6 +1067,38 @@ type AviHostPathPortPoolPG struct {
 	IngName       string
 }
 
+func (v *AviHostPathPortPoolPG) GetCheckSum() uint32 {
+	// Calculate checksum and return
+	v.CalculateCheckSum()
+	return v.Checksum
+}
+
+func (v *AviHostPathPortPoolPG) CalculateCheckSum() {
+	var checksum uint32
+	sort.Strings(v.Path)
+	sort.Strings(v.Host)
+	checksum = checksum + utils.Hash(utils.Stringify(v))
+	v.Checksum = checksum
+}
+
+func (v *AviHostPathPortPoolPG) GetNodeType() string {
+	// Calculate checksum and return
+	return "AviHostPathPortPoolPG"
+}
+
+func (v *AviHostPathPortPoolPG) CopyNode() AviModelNode {
```

Comment:
`AviHostPathPortPoolPG` is inner field of the `httpPolicyset` object.  will it be required? When will it be called?

Reply:
This function is no longer required/called hence I have removed it.